mrchecker-framework-modules/mrchecker-example-module/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <artifactId>mrchecker-test-framework</artifactId>
         <groupId>com.capgemini.mrchecker</groupId>
-        <version>2.4.1</version>
+        <version>2.4.2-SNAPSHOT</version>
     </parent>
     <artifactId>mrchecker-example-module</artifactId>
     <version>${version}</version>
```

Comment:
Line 52 (I'm unable to place a comment at that line) contains old core version.

Reply:
done